### DISCUSSION TITLE
Unable to create post or  discussion

### DISCUSSION THREAD

**Discussion by YonSci** [DNNCommunity/Dnn.CommunityForums#295] [Q&A]
I am new to DNN environment, i am able to set up Active Forums 07.00.06 module to my site.  But i dont know how  to create forums or discussion so that users can replay or comment on it or how to customize some default forum.  Do you have some tutorial on it? Thanks in Advance!

**Comment by Timo-Breumelhof** [comment]
Did you check out the Wiki?
https://github.com/DNNCommunity/Dnn.CommunityForums/wiki

**Reply by WillStrohl** [reply]
@YonSci As Timo suggests, there really shouldn't be much to it, since you've set it up already.  From there, it's a simple as clicking around.  If there is more to what's happening to you, you should update your question to include those details.  🤔
